stdlib/public/core/ContiguousArrayBuffer.swift

```diff
@@ -417,13 +417,16 @@ internal struct _ContiguousArrayBuffer<Element>: _ArrayBufferProtocol {
   }
 
 #if _runtime(_ObjC)
+  
   /// Convert to an NSArray.
   ///
   /// - Precondition: `Element` is bridged to Objective-C.
   ///
   /// - Complexity: O(1).
-  @inlinable
+  @usableFromInline
   internal __consuming func _asCocoaArray() -> AnyObject {
+    let bridgeWorks = _bridgeInitializedSuccessfully
```

Comment:
I'm assuming this is split out because you still want to trigger the initialization of `_bridgeInitializedSuccessfully` in no-assert builds. Might be good to put a comment to that effect, so some enthusiastic refactorer from the future doesn't decide to consolidate these two lines.

Reply:
I consolidated the debugAssert into the initializer :) less duplication, no chance of this mistake. Thanks for pointing it out!

Reply:
Why are we doing a let-binding rather than a method call? That would be clear at the use site that something is actually happening.

Reply:
accessing a global is pretty much the only way to do a dispatch_once in swift